org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -148,6 +152,28 @@ public SpaceMode DistanceSpaceMode
         [Tooltip("Speed for extending the moving button visuals when selected by a non-touch source.")]
         private float extendSpeed = 0.5f;
 
+        [field: SerializeField, Tooltip("Enables (true) or disables (false) the Canvas Rounded Rect on proximity only.")]
+        public bool EnableOnProximityOnlyCanvasRoundedRect { get; set; } = false;
```

Comment:
If we are passing in components here (i.e. FrontPlate)  I don't think we need to have these EnableOnProximityOnlyCanvasRoundedRect or EnableOnProximityOnlyFrontPlateRawImage fields.

If someone wanted to opt of the front plate disablement, they can leave this FrontPlate field empty

Reply:
Makes sense, I'll update accordingly, thx for the feedback.  Actually, since the new direction is to use an array of components in which the user decides which components to make dynamic based on proximity (as per udpated User Story) I will remove these fields.

Reply:
In the last commit I removed these fields as the new behavior is to enable/disable by proximity any Component in such array as per new requirements.  Resolving this conversation.